src/FirstInstall/FirstInstallInstructions.jsx

```diff
@@ -48,20 +47,109 @@ import { hideFirstSteps, selectedVersion } from '../Manager/managerReducer';
     const sampleDir = [zephyrDir, 'samples', 'basic'].join(sep);
-    const homeDir = os.homedir();
     const suggestedExample = 'blinky';
     const suggestExampleCmakelists = [sampleDir, suggestedExample, 'CMakeLists.txt'].join(sep);
     const boardName = 'nrf9160_pca10090';
     const suggestExampleBuildDir = [sampleDir, suggestedExample, `build_${boardName}`].join(sep);
     const exampleBoardDir = [zephyrDir, 'boards', 'arm', boardName].join(sep);
 
+    return (
+        <ul>
+            <li>
+                <b>CMakeLists.txt</b> - the location of
+                the <code>CMakeLists.txt</code> project file of the sample
+                that you want to work with<br />
+                <Ie /><code>{suggestExampleCmakelists}</code>
+            </li>
+            <li>
+                <b>Board Directory</b> - the location of the board description
+                of the board for which to build the project<br />
+                <Ie /><code>{exampleBoardDir}</code>
+            </li>
+            <li>
+                <b>Board Name</b> - the board name (select from the list that
+                is populated based on the board directory)<br />
+                <Ie /><code>{boardName}</code>
+            </li>
+            <li>
+                <b>Build Directory</b> - the directory in which to run the build
+                (automatically filled based on the board name, but you can
+                specify a different directory)<br />
+                <Ie /><code>{suggestExampleBuildDir}</code>
+            </li>
+            <li>
+                <b>Clean Build Directory</b> - select this option to ensure
+                that you are not building with an outdated build cache
+            </li>
+        </ul>
+    );
+};
+
+const NewSteps = () => (
+    <ul>
+        <li>
+            <b>nRF Connect SDK Release</b> - Select the NCS version that you
+            want to work with.<br />
+            The drop-down list contains the current version of all NCS
+            installation directories that SES knows about.
+        </li>
+        <li>
+            <b>nRF Connect Toolchain Version</b> - Select the version of the
+            toolchain that works with the selected NCS version.<br />
+            The drop-down list contains only toolchain versions that are
+            compatible with the selected NCS version.
+        </li>
+        <li>
+            <b>Projects</b> - Select the project that you want to work with.
+            <br />
+            The drop-down list contains a selection of samples and
+            applications from the sdk-nrf and sdk-zephyr repositories.
+            Select any of the checkboxes underneath to add the samples from
+            that area to the drop-down list. To add projects to the
+            drop-down list, for example, your own custom projects, click
+            {' '}<Dotdotdot /> and select the folder that contains the
+            projects that you want to add.
```

Comment:
Could shorten to:
```suggestion
            Select any of the checkboxes to add samples and applications from
            the respective area to the drop-down list. To add custom projects to the
            drop-down list, click
            {' '}<Dotdotdot /> and select the folder that contains the
            projects that you want to add.
```
I think that's sufficient here, and the docs have more info for users that want the details.